collectd_openstack/gnocchi/writer.py

```diff
@@ -100,4 +101,8 @@ def _send_data(self, metername, to_send, unit=None):
 
         # gnocchi samples
         payload = json.dumps([sample.to_payload() for sample in to_send])
-        self._sender.send(metername, payload, unit=unit)
+
+        if 'cuda' in str(metername):
```

Comment:
Re: the other comment, maybe it makes sense to name these `node` and `instance`? To differentiate between what they are actually being tied to. Can rename the senders then to `instance_sender` and `node_sender`.